lib/chercheville.ex

```diff
@@ -34,24 +24,34 @@ defmodule ChercheVille do
 
   Update your database schema. This will add a table named `cities`:
 
-      $ mix ecto.migrate -r ChercheVille.Repo
+      $ mix ecto.migrate
 
   A couple of mix tasks are available to populate the database. Each task takes
   a list of [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
   as arguments.
 
+  ## Importing data
+
   To fetch data files from geonames.org and store them locally:
 
       $ mix chercheville.fetch_data FR BE ES
 
   To load data from those files into our database:
 
-      $ mix chercheville.load_data FR BE ES
+      $ mix chercheville.import_data FR BE ES
+
+  ## Starting web service
+
+  Start the web server:
+
+      mix run --no-halt
+
+  Then visiting the http://localhost:4000 should show a list of available endpoints.
 
-  ## Searching for cities.
+  ## Calling Elixir functions directly
   
```

Comment:
Would it be worth putting here that we can run with iex with:
`
iex -S mix run --no-halt
`